pkg/scheduler/scheduler_test.go

```diff
@@ -946,7 +947,8 @@ func setupTestScheduler(queuedPodStore *clientcache.FIFO, scache internalcache.C
 		Profiles: profile.Map{
 			testSchedulerName: fwk,
 		},
-		client: client,
+		client:          client,
```

Comment:
Can we add unit tests for the cases of failure during binding?

Reply:
Sure, let me craft some tests to cover that.